symengine/tests/test_series_expansion.py

```diff
@@ -1,22 +1,23 @@
 from symengine.utilities import raises
 from symengine.lib.symengine_wrapper import (series, have_piranha, have_flint,
-    Symbol, Integer, sin, cos, exp, sqrt, E)
+                                             Symbol, Integer, sin, cos, exp,
+                                             sqrt, E)
```

Comment:
PEP 8 is more what you'd call guidelines than actual rules. So you can skip styles that are not appropriate